packages/shorebird_cli/test/src/commands/patch_command_test.dart

```diff
@@ -433,9 +452,6 @@ Please create a release using "shorebird release" and try again.
 
     test('throws error when release artifact cannot be retrieved.', () async {
       const error = 'something went wrong';
-      when(
```

Comment:
Why are these no longer needed?

Reply:
They are the default stubs so they were unnecessary